src/sentry/static/sentry/app/components/events/interfaces/threads.jsx

```diff
@@ -139,14 +135,20 @@ class Thread extends React.Component {
   };
 
   renderMissingStacktrace = () => {
+    const {crashed, errored} = this.props.data;
+    let message = 'No or unknown stacktrace';
+    if (crashed) {
+      message = 'Thread Crashed';
+    } else if (errored) {
+      message = 'Thread Errored';
+    }
+
     return (
       <div className="traceback missing-traceback">
         <ul>
           <li className="frame missing-frame">
             <div className="title">
-              <span className="informal">
-                {this.props.data.crashed ? 'Thread Crashed' : 'No or unknown stacktrace'}
-              </span>
+              <span className="informal">{t(message)}</span>
```

Comment:
I've seen the `No or unknown stacktrace` a few times, I found useful to point out the thread has no frames.

Reply:
`t()` only works on static strings